renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": [
+		":preserveSemverRanges",
+		"group:all",
+		"schedule:weekly",
+		":maintainLockFilesWeekly"
+	],
+	"separateMajorReleases": true,
```

Comment:
I'm not sure if this really works with `"group:all"`. I assume you want all patch/minor releases combined each week but major releases kept each in their own PR?

Reply:
Hi @rarkins,
just testing out at the moment but yes thats what i try to achieve :)